03_launch_mgmt_cluster.sh

```diff
@@ -39,14 +36,31 @@ source lib/ironic_basic_auth.sh
 # BMO and Ironic deployment functions
 # ------------------------------------
 
+function deployIronicAndOrBMO() {
+  export NAMEPREFIX=${NAMEPREFIX:-"baremetal-operator"}
+
+  IRONIC_DATA_DIR="${IRONIC_DATA_DIR:-/opt/metal3/ironic/}"
```

Comment:
```suggestion
  IRONIC_DATA_DIR="${IRONIC_DATA_DIR:-/opt/metal3/ironic}"
```
Don't include trailing slash by default.